src/plugins/js-eval/Dockerfile

```
@@ -7,9 +7,7 @@ RUN node_version=$(curl -sSL https://nodejs.org/download/release/ | awk '/>v/{ s
 RUN node --version
 
 # install shims, and acorn+nearley needed by engine262
-RUN npm i --prefix /run airbnb-js-shims@latest acorn@latest nearley@latest
-COPY vendor/engine262.js /run/node_modules/engine262.js
-
+RUN npm i --prefix /run airbnb-js-shims@latest acorn@latest nearley@latest https://github.com/engine262/engine262.git
```

```suggestion
RUN npm i --prefix /run airbnb-js-shims@latest acorn@latest nearley@latest engine262/engine262
```
github is assumed, and this way it can use github protocols instead of only https.